### DISCUSSION TITLE
After updating to v3.2.4 a value is missing.

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP_Buderus_KM271#47] [General]
I have installed update v3.2.4. The value for HK2 summer/winter is now missing.
Can you check that for once?

Thank you
Manfred

![buderus](https://github.com/dewenni/ESP_Buderus_KM271/assets/47242804/b5a34eaa-e0d0-4e9e-949e-4eef160be0d4)

**Comment by dewenni** [comment]
Hi Manfred,

that was already a discussion in the past. If the Logamatic is in "manual Mode", there is no valid signal for "summer/winter"

=> https://github.com/dewenni/ESP_Buderus_KM271/discussions/41#discussioncomment-6481985

regards
Sven

**Comment by Kenaschon** [comment]
Hm, but in the version buderus_km271_ota_update_v3.2.3.bin
it works perfect.

![buderus1](https://github.com/dewenni/ESP_Buderus_KM271/assets/47242804/405d39a5-e506-4b7f-8a5f-de73006f9fa1)

**Reply by dewenni** [reply]
Shure?
What happen when you set it to summer mode?
Does it show summer or winter?
If you are in manual mode, the flag for summer/winter seems to be invalid.
In the past it shows always winter when you are in manual mode, even it is "summer"

**Comment by Kenaschon** [comment]
Also in der Version v3.2.3 klappt das ohne Probleme. Dort erscheint immer das Richtige.

Ich kann Dir gerne Bilder hochladen.

Grüße

**Reply by dewenni** [reply]
Nochmal zur Erklärung.

Die Logamatik liefert einen Betriebswert `HK1_BW2_Sommer`. Dieser ist 1 wenn Sommer und 0 bei Winter ODER wenn die Betriebsart auf Manuell steht.
Das sieht man auch auf dem Display der Logamatik. Bei Sommer wird da eine Palme angezeigt. Auch die verschwindet wenn man die Betriebsart auf manuell stellt.

Bisher hatte ich die Anzeige Sommer/Winter einfach von dem Betriebswert `HK1_BW2_Sommer` gebildet.
Wenn man dann die Betriebsart auf manuell gestellt hat, wurde fälschlicherweise Winter angezeigt, weil das Bit für `HK1_BW2_Sommer` auch auf Null ging.

Das müsste bei dir auch so sein. Daher habe ich die Anzeige bei manueller Betriebsart jetzt weg genommen.

Alternativ könnte man es vielleicht noch mit der aktuellen Außentemperatur und dem Parameter für `Sommer_ab` noch selbst nachbilden.

**Comment by Kenaschon** [comment]
Ok, danke für die Erklärung. Könnte das man denn nachbilden?

Hattest du schon was mit der Außentemperatur gemacht?

**Reply by dewenni** [reply]
@Kenaschon 
Im nachten Update habe ich eine manuelle "Definition" von Sommer/Winter implementiert.
Wenn die Logamatic auf Manuell steht, dann wird der Status "Sommer/Winter" anhand der aktuellen Außentemperatur und dem Config Wert für "Sommer ab" gebildet.

**Comment by Kenaschon** [comment]
Danke.

**Reply by dewenni** [reply]
ist im aktuellen release candidate implementiert #80
